src/material/core/density/_all-density.scss

```diff
@@ -0,0 +1,10 @@
+@import '../theming/all-theme';
+
+// Includes all of the density styles.
+@mixin angular-material-density($config) {
```

Comment:
@jelbourn We denoted individual mixins with an underscore in `src/material`. Should we do this here? I'm asking because we are working in feature branch and the underscore might not be actually needed.

(Might be worth removing from the individual mixins too)